overlord/confdbstate/transaction.go

```diff
@@ -75,6 +84,7 @@ type marshalledTransaction struct {
 func (t *Transaction) MarshalJSON() ([]byte, error) {
 	return json.Marshal(marshalledTransaction{
 		Pristine:      t.pristine,
```

Comment:
I think this can be dropped now

Reply:
I'm not sure what you mean, Pristine is still used in the transaction

Reply:
Ahh, I was confused, I thought it was exported for the daemon but it seems to be marshaled to be persisted in the state, no?

Reply:
If that's the case, then LGTM

Reply:
that's right :) thanks for the review